rubrics/api/src/main/java/org/sakaiproject/rubrics/logic/model/Evaluation.java

```diff
@@ -102,6 +102,10 @@
     @Column(nullable = false)
     private EvaluationStatus status = EvaluationStatus.DRAFT;
 
+    @Enumerated
+    @Column(name = "evaluated_item_owner_type", nullable = false)
+    private EvaluatedItemOwnerType evaluatedItemOwnerType = EvaluatedItemOwnerType.USER;
```

Comment:
Since this is the value at instantiation should this be NONE here?

Reply:
Suppose so. I only added the NONE type as I needed the zero based ordinal to allow clean DB backfills in the update scripts, to be honest.

Reply:
Maybe change NONE to UNUSED?

Reply:
I removed the NONE enum  and made the column nullable, with a default on the bean of USER. This seems to make sense to me.